incubator/schema-registry/values.yaml

```diff
@@ -90,20 +90,31 @@ sasl:
 kafka:
   ## This is enabled only to allow installations of this chart without arguments
   enabled: true
-  imageTag: 4.0.1
+  ## Override kafka settings for default installations
   configurationOverrides:
     # Needed to run with 1 Kafka Broker
     offsets.topic.replication.factor: 1
+  ## Run only a single kafka broker by default
   replicas: 1
+  ## Do not enable kafka persistence by default. This setting is only acceptable in test
+  ## environments.
+  persistence:
+    enabled: false
```

Comment:
is a more sane value to enable persistence by default? see https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#Persistence